### PR TITLE
Add allow-all Hive security

### PR DESCRIPTION
**Pull request by kokosing**
Add allow-all Hive security

### CODE REVIEW

File: presto-docs/src/main/sphinx/connector/hive-security.rst [hunk 1]
```diff
@@ -34,6 +34,8 @@ Property Value                                     Description
                                                    To alter these privileges, use the :doc:`/sql/grant` and
                                                    :doc:`/sql/revoke` commands.
                                                    See :ref:`hive-sql-standard-based-authorization` for details.
+
+``allow-all``                                      No Authorization checks are enforced.
```
Comment (mosabua) [inline presto-docs/src/main/sphinx/connector/hive-security.rst:38]: authorization

Comment (mosabua) [inline presto-docs/src/main/sphinx/connector/hive-security.rst:38]: Is that really all the docs needed?

Comment (kokosing) [reply presto-docs/src/main/sphinx/connector/hive-security.rst]: Yes

**Comment by kokosing** [comment]
AC

**Review by mosabua — APPROVED** [review]
Doc lgtm

**Comment by kokosing** [comment]
Failed with: https://github.com/prestosql/presto/issues/5424

**Review by losipiuk — APPROVED** [review]
Some smoke test would be nice-to-have

**Pull request merged**